src/library_openal.js

```diff
@@ -1450,7 +1450,7 @@ var LibraryOpenAL = {
         if (srcLen > 0.0) {
           var frequency;
           for (var bufId in src.bufQueue) {
```

Comment:
hmm, is this even the right loop? these look like arrays of ints, so we should do `of` instead of `in` in modern JS. For backwards compatibility we can do `for (var i = 0; i < src.bufQueue.length; i++)` etc., but that's larger...

The current loops are confusing, but not horrible I guess.

Reply:
It does work, but agreed, if there are people who imbue arrays with custom properties, this has the problem of also iterating over them.

I am hesitant to go changing any behavior for this now though, since not focused on AL at the moment (and I think this is a bit deprecated-ish library anyway as it is non-multithreading friendly)